app/webpacker/components/Panel/pages/BannedCompetitorsPage/BannedCompetitorForm.jsx

```diff
@@ -27,7 +29,9 @@ export default function BanendCompetitorForm({
     }, () => {
       sync();
       closeForm();
-    }, { method: 'POST' });
+    }, { method: 'POST' }, (error) => {
+      setFormError(error);
+    });
```

Comment:
Just pass in `setFormError` as a callback reference?

Reply:
```suggestion
    }, { method: 'POST' }, setFormError);
```

Reply:
Done